definitions/divorce/json/CaseField/CaseField.json

```diff
@@ -942,6 +942,15 @@
     "FieldTypeParameter": "DivorceDocument",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "D8ConfidentialDocumentsUploaded",
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.